TargetTester.cpp

```diff
@@ -445,12 +445,10 @@ void TargetProtoTester::run_test(std::function<void(void)> fun,
 		fun();
 	}
 	catch (ETarget& err) {
-		status = err.status;
```

Comment:
Names of a class private fields are prefixed with underscore. I was inspired by [google c++ coding style](https://google.github.io/styleguide/cppguide.html#Variable_Names). I decided to make this field public, so there should be no underscore.